packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginActions.test.ts

```diff
@@ -73,21 +74,42 @@ describe('createPluginActions', () => {
       } = createPluginActions(uid, postToContainer, onUpdateState)
       setModalOpen(false)
 
-      expect(postToContainer).toHaveBeenLastCalledWith(
+      expect(postToContainer).toHaveBeenCalledWith(
         expect.objectContaining({
           event: 'toggleModal',
           status: false,
         } satisfies Partial<ModalChangeMessage>),
       )
 
       setModalOpen(true)
-      expect(postToContainer).toHaveBeenLastCalledWith(
+      expect(postToContainer).toHaveBeenCalledWith(
```

Comment:
This was changed during development for a reason that does not apply anymore. It does not really matter, but I kept it as-is.